zebra-consensus/src/chain.rs

```diff
@@ -1,128 +1,70 @@
-//! Chain state updates for Zebra.
-//!
-//! Chain state updates occur in multiple stages:
-//!   - verify blocks (using `BlockVerifier` or `CheckpointVerifier`)
-//!   - update the list of verified blocks on disk
-//!   - create the chain state needed to verify child blocks
-//!   - choose the best tip from all the available chain tips
-//!   - update the mature chain state on disk
-//!   - prune orphaned side-chains
-//!
-//! Chain state updates are provided via a `tower::Service`, to support
-//! backpressure and batch verification.
-
 #[cfg(test)]
```

Comment:
Hmm, is this `}` a GitHub display issue?

Reply:
I don't see a `}` (?) so it seems likely (?)